packages/app/src/overview/components/update-os-modals/downloading-update-modal/downloading-update-modal.component.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { DownloadingUpdateModalProps } from "App/overview/components/update-os-modals/downloading-update-modal/downloading-update-modal.interface"
+import { DownloadBar } from "App/overview/components/update-os-modals/downloading-update-modal/downloading-update-modal.styled"
+import { OSUpdateModal } from "App/overview/components/update-os-modals/os-update-modal"
+import { ModalMainText, RoundIconWrapper } from "App/ui/components/modal-dialog"
+import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"
+import Icon from "App/__deprecated__/renderer/components/core/icon/icon.component"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { convertBytes } from "App/__deprecated__/renderer/utils/convert-bytes"
+import formatDuration from "App/__deprecated__/renderer/utils/format-duration"
+import { intl } from "App/__deprecated__/renderer/utils/intl"
+import React from "react"
+import { defineMessages, FormattedMessage } from "react-intl"
+
+const messages = defineMessages({
+  muditaOsUpdateTitle: {
+    id: "module.overview.muditaOsUpdateTitle",
+  },
+  downloadingUpdateMessage: {
+    id: "module.overview.downloadingUpdateMessage",
+  },
+  downloadingUpdateDescriptionStarting: {
+    id: "module.overview.downloadingUpdateStarting",
+  },
+  downloadingUpdateDescriptionDownloading: {
+    id: "module.overview.downloadingUpdateDownloading",
+  },
+  downloadingUpdateDescriptionFinishing: {
+    id: "module.overview.downloadingUpdateFinishing",
+  },
```

Comment:
```suggestion
  downloadingUpdateDescriptionStarting: {
    id: "module.overview.downloadingUpdateDescriptionStarting",
  },
  downloadingUpdateDescriptionDownloading: {
    id: "module.overview.downloadingUpdateDescriptionDownloading",
  },
  downloadingUpdateDescriptionFinishing: {
    id: "module.overview.downloadingUpdateDescriptionFinishing",
  },
```

Reply:
Changed in https://github.com/mudita/mudita-center/pull/1041/commits/10c85ad257d39a6d26518f6e2b46aa5120ac049c